api-docs/cloud-load-balancers-v1/api-reference/methods/get-list-ciphers-v1.0-account-loadbalancers.rst

```diff
@@ -0,0 +1,102 @@
+|422                       |ImmutableEntity          |This fault is returned   |
+|                          |                         |when a user attempts to  |
+|                          |                         |modify an item that is   |
+|                          |                         |not currently in a state |
+|                          |                         |that allows              |
+|                          |                         |modification. For        |
+|                          |                         |example, load balancers  |
+|                          |                         |in a status of           |
+|                          |                         |PENDING_UPDATE,BUILD, or |
+|                          |                         |DELETED may not be       |
+|                          |                         |modified.                |
++--------------------------+-------------------------+-------------------------+
+|500                       |Load Balancer Fault      |The load balancer has    |
+|                          |                         |experienced a fault.     |
++--------------------------+-------------------------+-------------------------+
+|503                       |Service Unavailable      |The service is not       |
+|                          |                         |available.               |
++--------------------------+-------------------------+-------------------------+
+
+Request
+-------
+
+The following table shows the URI parameters for the request:
+
++--------------------------+-------------------------+-------------------------+
+|Name                      |Type                     |Description              |
++==========================+=========================+=========================+
+|{account}                 |String                   |The ID for the tenant or |
+|                          |                         |account in a multi-      |
+|                          |                         |tenancy cloud.           |
++--------------------------+-------------------------+-------------------------+
+|{loadbalancerid}          |String                   |The ID of the load       |
+|                          |                         |balancer.                |
+|                          |                         |                         |
++--------------------------+-------------------------+-------------------------+
+
+This operation does not accept a request body.
+
+Response
+--------
+
+
+The following table shows the body parameters for the response:
+
++--------------------------+-------------------------+-------------------------------+
+|Name                      |Type                     |Description                    |
++==========================+=========================+===============================+
+|cipherList                |String                   |A ``loadBalancers``            |
```

Comment:
The Description value says "A loadBalancers object.", can we add little more explanation here? For ex: List of comma separated cipher names that are defined on the loadbalancer.